browser/brave_profile_prefs.cc

```diff
@@ -377,6 +377,8 @@ void RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
                                 false);
   registry->RegisterBooleanPref(brave_shields::prefs::kAdBlockDeveloperMode,
                                 false);
+  registry->RegisterBooleanPref(
```

Comment:
We always register this pref, even if the feature is disabled, so FindPreference seems redundant, because an unset pref will still return false. Am I missing something?